make/photon/notary/builder_public

```diff
@@ -41,19 +27,18 @@ echo 'build the notary binary bases on the golang:1.7.3...'
 cp binary.Dockerfile $TEMP 
 cd $TEMP
 docker build -f binary.Dockerfile -t notary-golang $TEMP
+cp $TEMP/migrations binary
```

Comment:
migrations is a directory right? so it needs `cp -r`?

Reply:
sure